### ISSUE TITLE
Jump to the end of a document.

### ISSUE THREAD

**Issue by dong628** [GitJournal/GitJournal#592]
[Feature] There isn't any hot keys to jump to the end of a document, although it is really important for me, especially in a long markdown note ( such as my diary... ).

**Comment by vHanda** [comment]
This is easy to implement, but I'm afraid that adding another item in the menu would result in a lot more clutter. The sub-menu already has tons of operations.

How about instead of providing a way to jump, GitJournal remembers the position from last time and opens the document over there?

**Comment by dong628** [comment]
> This is easy to implement, but I'm afraid that adding another item in the menu would result in a lot more clutter. The sub-menu already has tons of operations.
> 
> How about instead of providing a way to jump, GitJournal remembers the position from last time and opens the document over there?

Great idea! Adding a toggle "remember the position" in the settings page might be the best solution.

Additionally, each time I switch to the reading mode ( render the Markdown note ) , I have to turn the page to the position where I have edited. It can be a necessary feature to switch the reading/editing mode and keep the position of the note.

**Comment by glow12121** [comment]
I agree this is an important feature for those working with longer documents.
